chain/chainservice.go

```diff
@@ -561,22 +566,34 @@ func (cs *ChainService) getNameInfo(qname string, blockNo types.BlockNo) (*types
 	} else {
 		stateDB = cs.sdb.OpenNewStateDB(cs.sdb.GetRoot())
 	}
-	return name.GetNameInfo(stateDB, qname)
+
+	ncs, err := state.GetNameAccountState(stateDB)
```

Comment:
what `ncs` stands for? Name Contract State?

Reply:
yes. scs is system contract state / ecs is enterprise contract state / ncs is name contract state.